cli/src/main/java/org/opentosca/toscana/cli/CliProperties.java

```diff
@@ -0,0 +1,99 @@
+package org.opentosca.toscana.cli;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+import org.apache.commons.io.FileUtils;
+
+public class CliProperties {
+
+    private static String filePath;
+    private static String dataPath = null;
+    private static final String dataPathWin = "/AppData/toscana";
+    private static final String dataPathNix = "/.toscana";
+    private static final String cliProp = "/cli.properties";
+    private static Properties properties = new Properties();
+    private static File file;
+    private static String OS = null;
+
+    /**
+     * Creates a cli.properties config file
+     */
+    private static void createProperties() {
+        try {
+            FileWriter writer = new FileWriter(file);
+            properties.setProperty("ip","http://localhost");
```

Comment:
i don't like the idea of splitting this in 3 parts. As i suggested in the original issue one property that includes the whole base url seems to be a way better solution.

Reply:
please use 127.0.0.1 instead of localhost. some hosts (especially some vm images /docker images) are not properly configured and localhost doesn't point to 127.0.0.1, this has often led to extrem irritation.